src/components/ComponentIndexPage/component-index.scss

```diff
@@ -148,3 +148,28 @@
   color: $text-01;
   margin-bottom: $spacing-06;
 }
+
+//----------------------------------------------------------------------------
+// Component Index Filter
+//----------------------------------------------------------------------------
+.component-index-filter-container {
+  margin-top: 12.5rem;
+}
+// Header
+.component-index-filter__header {
+  @include carbon--type-style('heading-01');
+  color: $text-01;
+}
+.bx--label  {
```

Comment:
Do we need to scope these selectors so they don't apply globally? 👀

Reply:
ah yes 🤦‍♀️